src/test/features/claim/routes/pay.ts

```diff
@@ -485,12 +491,43 @@ describe('Claim issue: post payment callback receiver', () => {
               .expect(res => expect(res).to.be.serverError.withText('Error'))
           })
 
+          it('should return 500 and render error page when feature toggle api fails', async () => {
+            draftStoreServiceMock.resolveFind(draftType, payServiceMock.paymentInitiateResponse)
+            idamServiceMock.resolveRetrieveServiceToken()
+            payServiceMock.resolveRetrieve('Success')
+            draftStoreServiceMock.resolveSave()
+            claimStoreServiceMock.resolveRetrieveClaimByExternalIdTo404HttpCode('Claim not found by external id')
```

Comment:
That does not look right. I think `expect(res).to.be.serverError.withText('Error')` is triggered because of that, not `featureToggleApiMock.rejectIsAdmissionsAllowed`